src/segments/wttr.bash

```diff
@@ -5,9 +5,18 @@ format=${SEGMENTS_WTTR_FORMAT:-'%p;%t;%w'}
 refresh_rate="${SEGMENTS_WTTR_REFRESH_RATE:-600}"
 
 segments::wttr_fetch_changes() {
-  curl \
+  web=$(curl -s \
     -H "Accept-Language: ${LANG%_*}" \
-    --compressed "wttr.in/${location}?format=${format}" | tr -d '\n' | tr ';' '\n'
+    --compressed "wttr.in/${location}?format=${format}"; exit $?)
+  rtn_code=$?
+
+  debug::log "wttr fetch return code $rtn_code"
+  debug::log "wttr fetch content $web"
```

Comment:
Debugging leftovers?

Reply:
Yes but I think we should keep it since wttr.in tends to fail in weird ways.